src/main.rs

```diff
@@ -64,6 +66,15 @@ fn main() -> Result<()> {
         (xcb::CW_OVERRIDE_REDIRECT, 1),
     ];
 
+    // TODO: Move the ``window_cmd_map`` into the configuration
+    let mut window_command = &wm::WindowCommand::Focus;
+    let mut window_cmd_map = HashMap::new();
+    window_cmd_map.insert("x", wm::WindowCommand::Kill);
+    window_cmd_map.insert("q", wm::WindowCommand::Float);
```

Comment:
Let's get rid of this `mut` and us the `HashMap::form([...])` syntax.

Reply:
Done, but I think it'll get made redundant when I add the cofiguration